config.json

```diff
@@ -18,6 +18,17 @@
 	    ]
 	},
 	"propagatedmount": "/mnt",
+	"mounts": [
+		{
+		"name": "KeyPath",
+		"description": "Path to SSH keys",
+		"source": "/tmp",
+		"destination": "/tmp/sshfs",
```

Comment:
@davidwilliamson If I am understand correctly, that identity file is accessible in all container instances that reference it. There doesn't seem to be a point where we check that a particular container is using a particular remote identity.

@cyli Do we have a way of accessing these keys from the secrets store? Like, can we mount the secret into the plugin?